src/test/java/redis/clients/jedis/tests/JedisTest.java

```diff
@@ -188,4 +188,56 @@ public void testBinaryBitfield() {
     }
   }
 
+  @Test
```

Comment:
One more, you can just use `jedis` without initialize yourself. Please refer other test methods in HashesCommandsTest.